### DISCUSSION TITLE
Debian 11 Installed, shows up, can't access...

### DISCUSSION THREAD

**Discussion by username-mendoza** [christgau/wsdd#167] [General]
goodmorning,

Installed this script, server name appears in (win 10) file explorer, but i cant enter shares by just clicking on it, throws error...

![image](https://user-images.githubusercontent.com/99207456/218105643-c17340ea-2a96-4bca-9f99-97553ba2d65c.png)

can access same server by entering ip address to address field... but that was always, even before wsdd...
  

The other server, runs Ubuntu (20.04, i think), everything just worked from very end of installation of it...

**Comment by christgau** [comment]
Wsdd is not concerned with connectivity or service availability. It only takes care that the device (the file server) and the service (samba) is discovered by other hosts, like those running Windows.

To solve your problem, make sure that Windows is able to resolve the announced hostname, which appears to be "server" in your case.
